hadoop-ozone/fault-injection-test/mini-chaos-tests/src/test/java/org/apache/hadoop/ozone/loadgenerators/LoadGenerator.java

```diff
@@ -22,6 +22,16 @@
  * Interface for load generator.
  */
 public abstract class LoadGenerator {
+  /**
```

Comment:
This is not a javadoc for `keyNameDelimiter`.

```suggestion
  /*
```